### DISCUSSION TITLE
How to  start proxy 5 times                         with different ports

### DISCUSSION THREAD

**Discussion by 1Megazone** [seevik2580/tor-ip-changer#78] [Q&A]
Dear Sir, 
I wanna to know how to make the following option/setting :

  '-m n' start multiple proxy n instances
            example:   ipchanger.exe -m 5
                        start proxy 5 times
                        with different ports
                        and generate list

I looking for you help and advice
thanks in advance

**Comment by seevik2580** [comment]
Hello,
create shortcut from your ipchanger.exe and change your Target: like in picture below, or use cmd.exe within ipchanger folder and just type "ipchanger.exe -m 5"
![image](https://user-images.githubusercontent.com/22060747/185514057-c3a75fba-1c2f-4e27-9993-5572a3cc0229.png)
